docs/learn-glossary.md

```diff
@@ -3,160 +3,175 @@ id: learn-glossary
-those tasks, and communicates task execution results back to the Temporal service.
-Worker services are developed, deployed, and operated by Temporal customers.
+A Worker, also known as a Worker service, is a service that hosts the [Workflow](#workflow) and [Activity](#activity) implementations.
+
+- The Worker polls the Temporal service for [Tasks](#task), performs those [Tasks](#task), and communicates [Task](#task) execution results back to the Temporal service.
+- Worker services are developed, deployed, and operated by Temporal customers.
 
 ### Workflow
 
-A fault-oblivious stateful function that orchestrates activities. A _Workflow_ has full control over
-which activities are executed, and in which order. A _Workflow_ must not affect
-the external world directly, only through activities. What makes workflow code
-a _Workflow_ is that its state is preserved by Temporal. Therefore any failure
-of a worker process that hosts the workflow code does not affect the workflow
-execution. The _Workflow_ continues as if these failures did not happen. At the
-same time, activities can fail any moment for any reason. Because workflow code
-is fully fault-oblivious, it is guaranteed to get notifications about activity
-failures or timeouts and act accordingly. There is no limit on potential workflow
-duration.
+A fault-oblivious stateful function that orchestrates activities.
+
+- A Workflow has full control over which [Activities](#activity) are executed, and in which order.
+- A Workflow must not affect the external world directly, only through [Activities]((#activity)).
+- What makes Workflow code a Workflow is that its state is preserved by Temporal. Therefore any failure of a [Worker](#worker) process that hosts the Workflow code does not affect the [Workflow Execution](#workflow-execution). The Workflow continues as if these failures did not happen. At the same time, [Activities](#activity) can fail any moment for any reason.
+- Because Workflow code is fully fault-oblivious, it is guaranteed to get notifications about [Activity](#activity) failures or timeouts and act accordingly.
+- There is no limit to the duration of a Workflow.
 
 ### Workflow Execution
 
-An instance of a _Workflow_. The instance can be in the process of executing
-or it could have already completed execution.
+An instance of a [Workflow](#workflow).
+
+- The instance can be in the process of executing or it could have already completed execution.
 
-### Workflow Id
+### Workflow ID
 
-A unique identifier for a _Workflow Execution_. Temporal guarantees the
-uniqueness of an Id within a namespace. An attempt to start a _Workflow_ with a
-duplicate Id results in an **already started** error.
+A unique identifier for a [Workflow Execution](#workflow-execution).
+
+- Temporal guarantees the uniqueness of an ID within a [Namespace](#namespace).
+- An attempt to start a [Workflow](#workflow) with a duplicate ID results in an **already started** error.
```

Comment:
... already started error if there is another open workflow execution. The behaviour of start when there is another already completed workflow execution depends on WorkflowIdReusePolicy flag. If it is set to ALLOW_DUPLICATE it is possible to start a new execution with the same id.